fasthttp-mysql/README.md

```diff
@@ -0,0 +1,135 @@
+# Movie List
+
+A sample application to get th list of movies from the API and display it in a list.
+
+## Prerequisites
+
+- [Go](https://go.dev/doc/install) 1.16 or higher
+- [Docker](https://docs.docker.com/engine/install/) for running mySQL database and keploy
+- API client like [Postman](https://www.postman.com/downloads/), [Insomnia](https://insomnia.rest/download/), [Hoppscotch](https://hoppscotch.io/) or cURL.
+
+## Start Keploy
+
+> Note that Testcases are exported as files in the repo by default
+
+### MacOS
+
+```shell
+curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_darwin_all.tar.gz" | tar xz -C /tmp
+
+sudo mv /tmp/keploy /usr/local/bin && keploy
+```
+
+### Linux
+
+```shell
+curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_amd64.tar.gz" | tar xz -C /tmp
+
+
+sudo mv /tmp/keploy /usr/local/bin && keploy
+```
+
+### Linux ARM
+
+```shell
+curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_arm64.tar.gz" | tar xz -C /tmp
+
+
+sudo mv /tmp/keploy /usr/local/bin && keploy
+```
+
+The UI can be accessed at <http://localhost:6789>
+
+## Start mySQL database
+
+```bash
+docker build -t ksql . && docker run -itd --rm -p 3306:3306 ksql
+```
+
+## Start Users-Profile sample application
+
+```bash
+git clone https://github.com/keploy/samples-go
+cd samples-go
+cd fasthttp-sql
+go get .
+export KEPLOY_MODE="record" 
+go run .
+```
+
+> export KEPLOY_MODE="record" changes the env to record test cases
+
+## Routes
+>
+> Sample Application Port: <http://localhost:8080>
+
+- POST /movie - Add a movie
+- GET /movie - Get last added movie
+- GET /movies - Get all movies
+<!-- - GET /movies?year={value} - Get all movies of year {value}
+- GET /movies?rating={value} - Get all movies of rating {value}
+- GET /movies?year={value}&rating={value2} - Get all movies of year {value} and rating {value2} -->
+
+## Generate Test Cases
+
+> Keploy Port: <http://localhost:6789/testlist>
+
+Start keploy server in a new terminal
+
+```bash
+keploy
+```
+
+To generate Test Cases, you need to make some API calls. It could be using Thunder Client, Postman Desktop Agent, or your preferred API testing tool.
+
+- **POST** movie in the database
+
+```bash
+curl --request POST \
+ --url http://localhost:8080/movie \
+ --header 'content-type: application/json' \
+ --data '{
+    "title":"Everything Everywhere All At Once",
+    "year":2022,
+    "rating":10
+}'
+```
+
+- **GET** movie in the database
+
+```bash
+curl --request GET \
+ --url http://localhost:8080/movie
+```
+
+- **GET** all movies in the database
+
+```bash
+curl --request GET \
+ --url http://localhost:8080/movies
+```
```

Comment:
Hi @JammUtkarsh I cannot see any mock generation happening when I tried running the app locally.

![Screenshot from 2023-03-23 12-47-15](https://user-images.githubusercontent.com/74777863/227131578-61cd58a1-b2d3-4029-ae1b-45892df7ac35.png)

![Screenshot from 2023-03-23 12-48-25](https://user-images.githubusercontent.com/74777863/227131612-4820609a-582d-4f96-8bc1-cdc0917c2ed8.png)

Reply:
Yes, due to a bug in fasthttp integration as mentioned [above](https://github.com/keploy/samples-go/pull/43#issuecomment-1479452497), the mocks are not being generated.